Use nf-validation on samplesheet (#616)
nf-core/ampliseq: docs/usage.md

### Input specifications

The input data can be passed to nf-core/ampliseq in three possible ways using the parameters `--input`, `--input_fasta`, or `--input_folder`.
The three parameters and input types are mutually exclusive.

- [Samplesheet input](#samplesheet-input) using `--input`: Tab-separated samplesheet
- [ASV/OTU fasta input](#asvotu-fasta-input) using `--input_fasta`: Fasta file with sequences to be taxonomically classified
- [Direct FASTQ input](#direct-fastq-input) using `--input_folder`: Folder containing zipped FastQ files.

Optionally, a metadata sheet can be specified for downstream analysis.

#### Samplesheet input

> The sample sheet file can be tab-separated (.tsv), comma-separated (.csv), or in YAML format (.yml/.yaml) that must have two to four columns/entries with the following headers:

Reviewer: ```suggestion
The sample sheet file can be tab-separated (.tsv), comma-separated (.csv), or in YAML format (.yml/.yaml) and can have two to four columns/entries with the following headers:
```
